corehq/apps/analytics/tests/test_hubspot.py

```diff
@@ -145,9 +145,9 @@ def setUpClass(cls):
         cls.blocked_commcare_user.save()
 
     def test_get_blocked_domains(self):
-        self.assertListEqual(
-            get_blocked_hubspot_domains(),
-            [self.blocked_domain.name, self.second_blocked_domain.name]
+        self.assertEqual(
+            set(get_blocked_hubspot_domains()),
+            {self.blocked_domain.name, self.second_blocked_domain.name}
```

Comment:
Offlined with @biyeun to confirm that the order of the result is not being tested.